flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/extraction/TypeInferenceExtractorTest.java

```diff
@@ -632,7 +633,9 @@ private static Stream<TestSpec> functionSpecs() {
                 TestSpec.forScalarFunction(FunctionHintTableArgScalarFunction.class)
                         .expectErrorMessage("Only scalar arguments are supported yet."),
                 TestSpec.forScalarFunction(ArgumentHintTableArgScalarFunction.class)
-                        .expectErrorMessage("Only scalar arguments are supported yet."));
+                        .expectErrorMessage("Only scalar arguments are supported yet."),
```

Comment:
You don't change the message, but I spotted the sentence is not grammatically correct. Do you think we could fix it?